libs/application/templates/public-debt-payment-plan/src/fields/PaymentPlan/PaymentPlan.tsx

```diff
@@ -180,7 +185,18 @@ export const PaymentPlan = ({ application, field }: FieldBaseProps) => {
         />
       )}
       {(isLoading || paymentPlanResults) && (
```

Comment:
Do we need this check ? 
Looks like this is being handled in the PaymentPlanTable component

Reply:
That's true. I will add this to the clickup card. I don't have time to remove it now. But would be great to get this live to get some sexy fixes.